aicsimageio/tests/readers/extra_readers/test_czi_reader.py

```diff
@@ -537,6 +542,98 @@ def test_multi_scene_aicsimage(
     )
 
 
+@pytest.mark.parametrize(
+    "filename, "
+    "first_scene_id, "
+    "second_scene_id, ",
+    [
+        (
+            "NoSceneName.czi",
```

Comment:
@toloudis from the GitHub issue I think the uploaded file was `NoSceneNames.czi`... If that is correct, @LegendLukaz I think this is a typo?

Reply:
Sorry for the long wait. I have edited that line, please re-run the workflow :)